### DISCUSSION TITLE
x-anchor: Get mouse position for manual style

### DISCUSSION THREAD

**Discussion by mauritskorse** [alpinejs/alpine#4121] [1. Help]
Is it possible to get the mouse position somehow within x-anchor when using manual styling?
Right now, $anchor only contains an x and y and it seems I cannot access $event within x-bind:style.

For example:
```html
<div 
  x-anchor.no-style="document.getElementById($id('tooltip'))"
  x-bind:style="{ position: 'absolute', top: $anchor.y +'px', left: $event.clientX + 'px' }" 
>
```

**Comment by SimoTod** [comment]
There's no $event in x-bind. You can have a property set to 0 in x-data that you update with `@mousemove` (which, being an event listener, has access to `$event`).
